include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -495,7 +495,7 @@ class argument_parser
      */
     argument_parser_meta_data info;
 
-private:
+protected:
```

Comment:
I thought we want to befriend a test fixture? 
(any way you forgot the `//!\privatesection`)